src/static/css/almanac.css

```diff
@@ -484,6 +515,7 @@ footer.alt-bg a:focus {
 footer {
   padding-top: 40px;
   padding-bottom: 40px;
+  z-index: 2;
```

Comment:
I don't think this is needed and think it's a little weird it goes behind the footer.